src/main/resources/migreringer/V1.0_0077__oppgave_id.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE oppgave_id_part(
+    id SERIAL PRIMARY KEY,
+    oppgave_ekstern_id VARCHAR(100) NOT NULL,
+    oppgavetype_ekstern_id VARCHAR(100) NOT NULL,
+    CONSTRAINT unique_oppgave_oppgavetype UNIQUE (oppgave_ekstern_id, oppgavetype_ekstern_id)
```

Comment:
Ha med oppgave_ekstern_versjon også i unique?

Reply:
_Kunne_ hatt med versjon. Ikke som erstatning for denne, men i tillegg: `CONSTRAINT UNIQUE (oppgave_ekstern_id, oppgave_ekstern_versjon)`.

Men jeg tenker versjon ikke er så relevant her, siden man uansett skal bruke siste versjon av en oppgave.

Reply:
Ah ja sant. Idempotenssjekk og unique constrainten blir jo håndhevet mot historikktabellen